x-pack/plugins/observability_ai_assistant/public/components/chat/chat_body.tsx

```diff
@@ -126,7 +142,55 @@ export function ChatBody({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [timelineContainerRef.current]);
 
-  if (connectors.loading || knowledgeBase.status.loading) {
+  if (!hasCorrectLicense) {
+    footer = (
+      <>
+        <EuiFlexItem grow className={incorrectLicenseContainer}>
+          <EuiPanel hasBorder hasShadow={false}>
+            <EuiFlexGroup
+              direction="column"
+              alignItems="center"
+              justifyContent="center"
+              className={incorrectLicenseContainer}
+            >
+              <EuiTitle>
+                <h2>{UPGRADE_LICENSE_TITLE}</h2>
+              </EuiTitle>
+              <EuiText color="subdued">
+                <p>
+                  {i18n.translate('xpack.observabilityAiAssistant.incorrectLicense.body', {
+                    defaultMessage: 'You need a Platinum license to use the Elastic AI Assistant.',
+                  })}
+                </p>
+              </EuiText>
+              <EuiButton fill>
+                {i18n.translate(
+                  'xpack.observabilityAiAssistant.incorrectLicense.subscriptionPlansButton',
+                  {
+                    defaultMessage: 'Subscription plans',
+                  }
+                )}
+              </EuiButton>
+              <EuiButtonEmpty>
+                {i18n.translate('xpack.observabilityAiAssistant.incorrectLicense.manageLicense', {
+                  defaultMessage: 'Manage license',
+                })}
+              </EuiButtonEmpty>
```

Comment:
What do these buttons do? (Might also be good to extract into a separate component)

Reply:
Fixed